test/browser/browser.sh

```diff
@@ -37,6 +37,11 @@ if grep -q 'ID=.*fedora' /etc/os-release; then
     dnf install -y tcsh
 fi
 
+# On CentOS Stream 8 the cockpit package is upgraded so the file isn't touched.
+if grep -q 'PRETTY_NAME="CentOS Stream 8"' /etc/os-release; then
```

Comment:
Hmm, this isn't really specific to CentOS, just the way images are built. At this point we can't tell any more if the test preparation installed or upgraded cockpit. I'd say, just check if the file doesn't exist

Reply:
Right, good point!